editor/src/templates/editor.tsx

```diff
@@ -309,7 +310,10 @@ export class Editor {
         // Ensure we have the correct theme set in VS Code
         void sendSetVSCodeTheme(getCurrentTheme(userState))
 
-        void isAuthenticatedWithGithub(loginState).then((authenticatedWithGithub) => {
+        void updateUserDetailsWhenAuthenticated(
+          this.boundDispatch,
+          isAuthenticatedWithGithub(loginState),
+        ).then((authenticatedWithGithub) => {
```

Comment:
Does this imply we should also be updating the code here? https://github.com/concrete-utopia/utopia/blob/8c7722e5b8dc9bb2d36d12e3bb45a4aec5bc90c9/editor/src/components/editor/server.ts#L393-L400

Reply:
Now that you've merged that change, yeah! :)

Reply:
Fixed in bc55a5974.